### ISSUE TITLE
Can't find any redstone in 0.61

### ISSUE THREAD

**Issue by EddyBuhler** [AshenGaming/potentia-reborn#16]
Tried digging for hours in the overworld and mining world. Tried going creative and setting several mining turtles to do the work for me. Not a single redstone drop to be had anywhere.

Basically, can't do the very second quest (I'm playing on peaceful) since that requires one redstone dust to build a transfer pipe.

**Comment by AshenGaming** [comment]
Hi, sorry about the issue. I had thought that I fixed this issue in this release, but I may have forgotten to update that config file. I'll post a solution for you tomorrow morning that won't require you to restart your world.

**Comment by AshenGaming** [comment]
The current solution is to go into the CoFH world config folder (Located at Curse/Instances/PotentiaReborn/config/cofh/world) and open config.cfg and change
`ReplaceStandardGeneration=false` to `ReplaceStandardGeneration=true` as well as 
`RetroactiveGeneration=false` to `RetroactiveGeneration=true`. Once you've done this, save the file, and open the `00_minecraft.json` file. Once you've done this, locate the code blocks that look like this:

 ```"coal": {
			"distribution": "uniform",
			"generator": {
				"block": "coal_ore",
				"material": "minecraft:stone",
				"cluster-size": 17
			},
			"cluster-count": 20,
			"min-height": 0,
			"max-height": 128,
			"retrogen": false,
			"biome": "all",
			"dimension": {
				"restriction": "blacklist",
				"value": [
					-1,
					1
				]
			}
```
and find change the `"retrogen": false` to `"retrogen": true` for each of the vanilla ores. Once you've done this, save the file and launch the game. Load up your current world, and you'll notice it start to do some retrogeneration. After this has complete (it should tell you, if not, just leave the game active for a while) you can open the config.cfg file from earlier and change `RetroactiveGeneration=true` to `RetroactiveGeneration=false`. And that should fix your problem. I'll make sure to fix this issue properly in the next version of the pack.

**Comment by EddyBuhler** [comment]
No problem, I can wait. :-)

**Comment by AshenGaming** [comment]
This has been fixed in 0.7.
